vertx-quickstart/pom.xml

```diff
@@ -113,14 +115,9 @@
                     <name>native</name>
                 </property>
             </activation>
-            <properties>
-                <quarkus.package.type>native</quarkus.package.type>
-                <!-- <quarkus.native.enable-https-url-handler>true</quarkus.native.enable-https-url-handler> -->
-            </properties>
             <build>
                 <plugins>
                     <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
I would likely leave these as IntellJ shows false errors when they are missing

Reply:
Hum... @ia3andy I believe code.quarkus.io does not contain the groupId. Is it something we should fix there too?

Reply:
I think there is an open issue about that somewhere...

Reply:
Yeah it's been fixed, let me find the commit

Reply:
https://github.com/quarkusio/quarkus/commit/508639444b4f7da9711382136cc637b62037b4a0